gulpfile.js

```diff
@@ -24,28 +27,38 @@ const paths = {
 
 function styles() {
 	return src(paths.src.sass + 'MAIN.scss')
-	.pipe(sourcemaps.init())
-	.pipe(sass().on('error', sass.logError))
-	.pipe(rename('ubcclf.css'))
-	.pipe(autoprefixer({
-		browsers: ['last 6 versions']
-	}))
-	.pipe(sourcemaps.write('./'))
-	.pipe(dest(paths.dist.css))
-}
+		.pipe(sass().on('error', sass.logError))
+		.pipe(rename(paths.dist.css))
+		.pipe(dest(paths.dist.path))
+};
+
+function prefix() {
```

Comment:
I wonder if this wouldn't be better named as `autoprefix` in case we ended up with a function that prefixed classes or something?

Reply:
Sure, I can rename it to autoprefix